cmd/storagenode/main.go

```diff
@@ -88,6 +105,59 @@ func cmdSetup(cmd *cobra.Command, args []string) (err error) {
 		filepath.Join(setupCfg.BasePath, "config.yaml"), overrides)
 }
 
+func cmdDiag(cmd *cobra.Command, args []string) (err error) {
+	diagCfg.BasePath, err = filepath.Abs(diagCfg.BasePath)
+	if err != nil {
+		return err
+	}
+
+	// check if the directory exists
+	_, err = os.Stat(diagCfg.BasePath)
+	if err != nil {
+		fmt.Println("Storagenode directory doesn't exist", diagCfg.BasePath)
+		return err
+	}
+
+	// open the sql db
+	dbpath := filepath.Join(diagCfg.BasePath, "piecestore.db")
+	db, err := psdb.Open(context.Background(), "", dbpath)
+	if err != nil {
+		fmt.Println("Storagenode database couldnt open:", dbpath)
+		return err
+	}
+
+	//get all bandwidth aggrements entries
+	bwAgreements, err := db.GetBandwidthAllocations()
+	t := table.NewWriter()
```

Comment:
Can't we just use tabwriter here? Or in other words, have you vetted the dependencies of pretty/table for security etc. and do we gain something significant from using it.

Reply:
scanned thru the code in this package. on a quick scan, the packages imported are basic golang supported packages

Reply:
Can we just use tabwriter?

Reply:
modified to use tab writer